documents/2018-arsmote-protokoll.tex

```diff
@@ -0,0 +1,94 @@
+\documentclass[a4paper,11pt,oneside]{article}
+\usepackage[swedish]{babel}
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc}
+\usepackage{fancyhdr}
+\usepackage{verbatim}
+\usepackage{graphicx}
+\usepackage{booktabs}
+\pagestyle{fancy}
+\fancyfoot{}
+\lhead{Årsmötesprotokoll 2018}
+\rhead{2018--02--19}
+\fancyfoot{}
+\fancyfoot[LE, RO]{\thepage}
+
+\begin{document}
+
+\section*{Årsmöte IX 2018}
+
+\section{Öppnande av mötet}
+Elis öppnar mötet 19:06.
+
+\section{Val av mötsordförande}
+Mötet väljer Elis Hirwing till mötesordförande.
+
+\section{Val av mötessekreterare}
+Mötet väljer Michael Stahre till mötessekreterare.
+
+\section{Val av justeringspersoner tillika rösträknare}
+Mötet väljer Kim Lindberger och Erik Welander till justeringspersoner tillika rösträknare.
+
+\section{Fråga om mötet är stadgeenligt kallat}
+Mötet finner att mötet är stadgeenligt kallat.
+
+\section{Verksamhetsberättelse}
+Verksamhetsberättelsen föredras och mötet beslutade att lägga till punkten om
```

Comment:
Ska det stå "förendras"?

Reply:
Nope, `föredras` som i att man `föredrar` något :)

Reply:
Right, bra. :) Tänkte att det kunde vara antingen eller eftersom vi också förändrade den. ;)

Reply:
Annars kan vi alltid smyga in ett n så vi har en förnedrad verksamhetsberättelse ;)

Reply:
Haha, ja, det är väl när vi har roast:at oss själva och vad vi har lyckats med ordentligt på årsmötet, kanske :p

Reply:
"beslutade att" borde nog vara "beslutar att"